### PR TITLE
add placeholder urls

### PR DESCRIPTION
**Pull request by witmicko**
JIRA: https://issues.jboss.org/browse/AEROGEAR-2728

Added docs placeholders to be linked to by client apps

### CODE REVIEW

**Review by JameelB — APPROVED** [review]

**Comment by JameelB** [comment]
Dockerfile needs to be regenerated

**Comment by ghost** [comment]
@witmicko LGTM + new Dockerfile

File: apb.yml [hunk 1]
```diff
@@ -11,6 +11,11 @@ metadata:
   longDescription: "An APB deploying the AeroGear UnifiedPush Server with persistent storage and authentication through Openshift"
   imageUrl: "https://pbs.twimg.com/profile_images/1794440005/aerogear_icon-1_400x400.png"
   documentationUrl: "https://github.com/aerogearcatalog/unifiedpush-apb/blob/master/docs/modules/ups_openshift/index.asciidoc"
+  sdkDocumentation:
```
Comment (maleck13) [inline apb.yml]: these can't be parsed in the broker in this format. may need to change to be something like 
```
- android-sdk-docs
- ios-sdk-docs
```

**Review by maleck13 — APPROVED** [review]

File: apb.yml [hunk 2]
```diff
@@ -17,6 +17,10 @@ metadata:
     - "docker.io/centos/postgresql-96-centos7:9.6"
     - "docker.io/aerogear/ups-config-operator:master"
     - "docker.io/openshift/oauth-proxy:v1.1.0"
+  sdk-docs-android: "http://docs.aerogear.org"
```
Comment (maleck13) [inline apb.yml:20]: are these the only urls we have? The url doesn't seem to work. Are there more specific urls that could be used?

**Pull request merged**